Add `The osu! YouTube legacy of ReeseWasHere` news post (#9477)
ppy/osu-wiki: news/2023/2023-06-02-the-osu-legacy-of-reesewashere.md

But what the community didn't know is that Reese's uploads were about to come to a screeching halt.

![](/wiki/shared/news/2023-06-02-osu-legacy-of-reesewashere/tweet.png)

On January 30th 2016, Reese officially announced via a now deleted [Reddit post](https://www.reddit.com/r/osugame/comments/43c2pk/future_of_the_reesewashere_youtube_channel/) on r/osugame and a [tweet](https://twitter.com/ReeseWasHere/status/693250557419593729) that the channel will no longer be active. I asked him if there was anything else that affected his decision to quit, besides getting a full-time job:

> ...after I graduated and started working I didn't really have much time to play osu! anymore, and I especially didn't have time to record liveplays. I wasn't really into the idea of just uploading videos of replay files, and I saw other channels were filling that gap, so I just stopped uploading.

> To my question of whether he still keeps up with the community, he responded that he participated in r/place last year and has kept up with the development of osu!(lazer), and is eagerly waiting for it to be the new main client. It was suggested that he will return at some point, but that the contributions would more likely be via coding, rather than uploading videos once again.

Reviewer: ```suggestion
To the question of whether he still keeps up with the community, he responded that he participated in r/place last year and has kept up with the development of osu!(lazer), eagerly waiting for it to be the new main client. He also suggested that he may return as a player at some point, but that any more contributions would likely happen via code, rather than uploading videos once again.
```

> I hope that this write-up hit older players with a wave of nostalgia as to how things used to be. I also hope that newer players had fun learning about this small, but meaningful period in osu! history, and how it has paved the way for the likes of [CPOL](https://www.youtube.com/@cpol_v1) to continue carrying the torch for replay channels on YouTube.

Reviewer: ```suggestion
---

Hopefully this write-up hit older players with a wave of nostalgia for how things used to be. I also hope that newer players had fun learning about this small, yet meaningful period in osu! history, and how it has paved the way for the likes of [CPOL](https://www.youtube.com/@cpol_v1) to continue carrying the torch for replay channels on YouTube.
```

> Even though Reese is no longer holding the camera, his legacy will always live in the community in some capacity.

Reviewer: more preposition variety

```suggestion
Even though Reese is no longer holding the camera, his legacy will always live with the community in some capacity.
```

> *You can view the full text interview I did with him via [this link](https://gist.github.com/0x84f/61c96e7c18a77eccad7e543674bbf0e9). Thank you [Tanza3D](https://osu.ppy.sh/users/10379965) for the news post banner!*

Reviewer: ```suggestion
The full [text interview](https://gist.github.com/0x84f/61c96e7c18a77eccad7e543674bbf0e9) is available for your perusal. Thank you [Tanza3D](https://osu.ppy.sh/users/10379965) for the news post banner!
```
